ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml

```diff
@@ -0,0 +1,67 @@
+base_images:
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
```

Comment:
I think should define the golang image within the repo [like we do for WMCO](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml#L7)

Reply:
Should a separate PR be open to reflect this change in other branches?

Reply:
A change like this can be a follow up PR. My suggestion would be to not make this change in this PR when enabling testing for the community branch.

Reply:
Will open up a follow-up PR once this merges